core/arch/x86/decode.c

```diff
@@ -2148,6 +2162,202 @@ decode_predicate_from_instr_info(uint opcode, const instr_info_t *info)
     return DR_PRED_NONE;
 }
 
+opnd_size_t
+decode_get_input_size_from_opcode(int opcode, bool evexw)
```

Comment:
This can be static to this file.  No need to export in a header: not used anywhere else.

Reply:
Done.